Local lists in HASSIL (#2030)
home-assistant/developers.home-assistant: docs/voice/intent-recognition/template-sentence-syntax.md

will match sentences such as "play the white album by the beatles". The `PlayAlbum` intent will have an `album` slot with "the white album " (note the trailing whitespace) and an `artist` slot with "the beatles".

#### Local lists

> Sometimes you don't need a slot list that would be available for all intents and sentences, so you can define one locally, making it usable only in the context of the intent data (i.e. collection of sentences) where it was defined. For example:

Reviewer: ```suggestion
Sometimes you don't need a slot list available for all intents and sentences, so you can define one locally, making it usable only in the context of the intent data (like a collection of sentences) where it was defined. For example:
```
